apps/accounts/views.py

```diff
@@ -115,6 +117,19 @@ def post(self, request, format=None):
         user = User.objects.get(email=email)
         user.provider = "local"
         token = OTP_DummyToken(user,"verify")
+
+        # Add an entry in the tbl_user_profile with dummy data
+        dummy_data = {
+            "user_id": user.user_id.hex,
+            "name" : user.name,
+            "email": user.email,
+            "user_type": user.user_type,
+            "about": "N/A"
+        }
```

Comment:
can we about as null

Reply:
Replaced the hardcoded value "N/A" with `None`